Project.toml

```diff
@@ -7,7 +7,9 @@ version = "0.1.4"
 
 [deps]
 AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
+StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
 AbstractMCMC = "2, 3"
+StatsBase = "0.33.4"
```

Comment:
What was the reason for adding the StatsBase dependency? Was it missing?

Reply:
Ah, hm. TBH I wouldn't have merged this yet... It is an accidential remainder from earlier things I tried.  

But still, this package would specify `StatsBase.sample` as part of the interface, so it probably should depend on and reexport that, right?

Reply:
`StatsBase.sample` is reexported by AbstractMCMC, so it's not necessary to depend on StatsBase directly to use and/or reexport `StatsBase.sample`. In fact, it might be easier to manage dependencies and versions if AbstractPPL does not depend on it if this would be the only reason.

Reply:
Cool, then I'll get rid of it.